PokeAlarm/Utils.py

```diff
@@ -128,6 +128,18 @@ def get_team_id(team_name):
     return get_team_id.ids.get(name)
 
 
+# Returns the types of a move when requesting
+def get_move_type(move_id):
+    if not hasattr(get_move_type, 'info'):
+        get_move_type.info = {}
+        file_ = get_path('data/move_info.json')
+        with open(file_, 'r') as f:
+            j = json.loads(f.read())
+        for id_ in j:
+            get_move_type.info[int(id_)] = j[id_]['type']
+    return get_move_type.info.get(move_id, 'unkn')
```

Comment:
We should hardcode this Unknown using the unknown class

Reply:
Do you want me to do the same for the rest of the Move Data while I'm in here or submit that via another PR?

Reply:
That can be in another PR - I think that we need to change our locales change to a singleton now that we removed multiprocessing. There's no reason to keep an identical copy of the mappings per manager.

Reply:
I also think a lot of the functions in `Utils.py` really need to be moved to `MonUtils.py` so that can be another cleanup PR